libs/service-portal/documents/src/screens/Overview/Overview.tsx

```diff
@@ -27,6 +27,8 @@ import isEqual from 'lodash/isEqual'
 import { ValueType } from 'react-select'
 import DocumentCard from '../../components/DocumentCard/DocumentCard'
 import { defineMessage } from 'react-intl'
+import { documentsSearchDocumentsInitialized } from '@island.is/plausible'
+import { useLocation } from 'react-use'
```

Comment:
Is there a reason for using `react-use` instead of `react-router(-dom?)`?
Since the location history is already tracked with `react-router` in the service-portal.

Reply:
not really, just used what was already there. Ill use the react-router instead.

Reply:
Fixed also removes the need for a check of pathname, good point @darrikonn